sdk/clientcore/core/src/main/java/io/clientcore/core/instrumentation/logging/LogLevel.java

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package io.clientcore.core.instrumentation.logging;
+
+import java.util.HashMap;
+import java.util.Locale;
+
+/**
+ * Enum which represent logging levels used.
+ */
+public enum LogLevel {
+    /**
+     * Indicates that no log level is set.
+     */
+    NOTSET(0, "0", "notSet"),
+
+    /**
+     * Indicates that log level is at verbose level.
```

Comment:
```suggestion
     * Indicates that the log level is at verbose level.
```
If you take this apply to the rest of the enum values